openbas-api/src/main/java/io/openbas/service/ScenarioService.java

```diff
@@ -456,8 +456,20 @@ public void exportScenario(
     // Add Injects
     objectMapper.addMixIn(Inject.class, Mixins.Inject.class);
     scenarioFileExport.setInjects(scenario.getInjects());
-    scenarioTags.addAll(
-        scenario.getInjects().stream().flatMap(inject -> inject.getTags().stream()).toList());
```

Comment:
Did you face any issues on using direclty getTags() instead of .getTags().stream()).toList() ?
I am worried